python/Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM python:3.7.3-slim as base
+
+ENV VERSION=0.0.1
+
+ENV PYTHONUNBUFFERED=true
+ENV PYTHONDONTWRITEBYTECODE=true
+
+WORKDIR /src/app
+
+COPY . .
```

Comment:
unrelated to grammars, but this is an anti-pattern as any change to any file requires `pip install` to run again, invalidating the layer cache